content/en/docs/setup/additional-setup/config-profiles/index.md

```diff
@@ -9,10 +9,11 @@ test: n/a
 ---
 
 This page describes the built-in configuration profiles that can be used when
-[installing Istio](/docs/setup/install/istioctl/).
-The profiles provide customization of the Istio control plane and of the sidecars for the Istio data plane.
-You can start with one of Istio’s built-in configuration profiles and then further customize the configuration for
-your specific needs. The following built-in configuration profiles are currently available:
+[installing Istio](/docs/setup/install/istioctl/). The profiles provide
```

Comment:
The named profiles below are instances of the IstioOperator CRD. Can you clarify how what you proposed would be different?

Reply:
Mainly related to review comments I had when I was writing the new multicluster docs. Sven suggested using full CRDs rather than pre-canned profiles. But I guess documenting the profiles that do exist still makes sense.

Reply:
Gotcha. Shipping the YAML for each profile instead of baking them into istioctl makes sense to me. More explicit and guides the user towards managing their config declaratively. Defaulting would be interesting. For example, does a profile inherit defaults from the current version istioctl or another YAML file. I believe we change defaults across releases which complicates things a bit.

Reply:
FWIW my preference is still to have profiles as they provide sets of defaults, but the documentation should use a CR to choose the profile rather than using command-line flags. Sets them up for declarative management of the config but starts with a small CR instead of a long one.

Reply:
This entire conversation is outside of the scope of this PR. Thus, I'm resolving it.